src/__tests__/byDisplayValue.test.js

```diff
@@ -100,3 +100,34 @@ test('findBy queries work asynchronously', async () => {
   await expect(findByDisplayValue('Display Value')).resolves.toBeTruthy();
   await expect(findAllByDisplayValue('Display Value')).resolves.toHaveLength(1);
 }, 20000);
+
+test('all queries should respect accessibility', async () => {
+  const Comp = () => (
+    <View>
+      <View accessibilityElementsHidden>
+        <TextInput value="test_01" />
+      </View>
+      <TextInput value="test_02" />
+    </View>
+  );
+
+  const {
+    findAllByDisplayValue,
+    findByDisplayValue,
+    getAllByDisplayValue,
+    queryAllByDisplayValue,
+    getByDisplayValue,
+    queryByDisplayValue,
+  } = render(<Comp />, { respectAccessibility: true });
+
+  await expect(findAllByDisplayValue('test_01')).rejects.toBeTruthy();
```

Comment:
consider `using rejects.toEqual(...)`

Reply:
nit: pls sort queries by they typical sort order: gets, query, find. Having them randomly shuffled raises question why they are in this particular order.